internal/provider/provider.go

```diff
@@ -102,6 +103,7 @@ func NewProvider(
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "Path to a CA certificate file to validate the server's certificate.",
+				DefaultFunc: schema.EnvDefaultFunc(api.EnvVaultCACert, ""),
```

Comment:
Curious what motivates adding this default func back to the SDKv2 provider?

Reply:
That was mistakenly added back. Will remove it. Thanks!